bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -255,6 +263,42 @@ def test_get_user_operator_status(self):
         assert response.status_code == 200
         assert response.json()['status'] == user_operator.status
 
+    def test_get_user_operator_data_industry_user(self):
```

Comment:
I went to add some new tests for that conditional I added and noticed we didn't have tests for this API route. I wrote 3 separate tests, one for CAS user and two for industry users. One where the industry user has a `user_operator` associated with them and one without.